features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/default/js/logic/asset.tag.edit.js

```diff
@@ -43,7 +43,7 @@ $(function () {
             type: 'GET',
             success: function (response) {
 
-                var tags = JSON.parse(response);
+                var tags = response;
```

Comment:
removed redundant variable `tags` in commit 4b1e692bd9a814ccd340a052abca7550c90f15c2 of PR #240